assets/lumiext/shaders/material/honey.frag

```diff
@@ -0,0 +1,17 @@
+#include frex:shaders/api/sampler.glsl
+#include frex:shaders/api/fragment.glsl
+#include lumiext:shaders/internal/frag.glsl
+
+/******************************************************
+  lumiext:shaders/material/slime.frag
+******************************************************/
+
+void frx_startFragment(inout frx_FragmentData data) 
+{
+#ifdef LUMI_PBRX
+  pbr_metallic = 0.3;
+  pbr_roughness = 0.4 - frx_luminance(data.spriteColor.rgb) * 0.7;
```

Comment:
This can produce negative roughness which results in undefined behavior. Did you mean to put `pbr_roughness = 0.0;`?

Reply:
I have no real idea about why the part after `0.4` is there, it was there in metalic, so its is there, if removing it will have no real visual difference, I can change it

Reply:
How did you end up with these shaders?

Reply:
Modifying the iron pbr shader. The bevels are there on purpose, makes the more translucent edges look better.